Makefile

```diff
@@ -8,7 +8,7 @@ INSTALL_DIR      ?= /usr/local
 vendor:
```

Comment:
When are you running it out of /usr/local? Not sure what this is in reference to.